src/components/molecules/NavBar/components/NormalRadio/NormalRadio.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useMemo } from "react";
+import { OverlayTrigger, Popover } from "react-bootstrap";
+import classNames from "classnames";
+
+import { generateId } from "utils/string";
+
+import { RadioModal } from "components/organisms/RadioModal/RadioModal";
+
+import { ButtonNG } from "components/atoms/ButtonNG";
+
+import "./NormalRadio.scss";
+
+interface NormalRadioProps {
+  volume: number;
+  setVolume: (value: ((prevState: number) => number) | number) => void;
+  title?: string;
+  onEnableHandler: () => void;
+  radioPlaying: boolean;
+  defaultShow: boolean;
+}
+
+export const NormalRadio: React.FC<NormalRadioProps> = ({
+  title,
+  defaultShow,
+  onEnableHandler,
+  radioPlaying,
+  volume,
+  setVolume,
+}) => {
+  const id = useMemo(() => generateId("NormalRadio-popover"), []);
+
+  const popoverClasses = classNames({
+    "NormalRadio NormalRadio--on": radioPlaying && volume,
+    "NormalRadio NormalRadio--off": !(radioPlaying && volume),
+  });
```

Comment:
```suggestion
  const popoverClasses = classNames("NormalRadio", {
    "NormalRadio--on": radioPlaying && volume,
    "NormalRadio--off": !(radioPlaying && volume),
  });
```

Reply:
B.E.M. 
Should always have B(lock) at the root, not start with M(odifier). 
This provides predictability for future uses.

For example, a top-down structure like in the provided images helps provide adequate styles in parent SCSS files without resulting to class injections that cause exceptionalism and anachronism in the child components. 
It may also become predictable enough that something similar like custom CSS theming can also be supported.

![sparkle-bem-02](https://user-images.githubusercontent.com/79229621/140737546-6a962994-696e-4134-abd1-9fa656304447.png)
![sparkle-bem-03](https://user-images.githubusercontent.com/79229621/140737541-43eb8f68-65d1-4bd8-a63d-8e075f4d1729.png)

Reply:
For this change, please first provide a reference to https://github.com/sparkletown/sparkle/blob/staging/docs/Questions.md (as per this precedent https://github.com/sparkletown/sparkle/pull/2219#discussion_r713837478 )